swift/ql/src/queries/Security/CWE-312/CleartextLogging.qhelp

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+Sensitive information that is logged unencrypted is accessible to an attacker
+who gains access to the logs.
+</p>
+</overview>
+
+<recommendation>
+<p>
+Ensure that sensitive information is always encrypted or obfuscated before being
+logged.
+</p>
+
+<p>
+In general, decrypt sensitive information only at the point where it is
+necessary for it to be used in cleartext.
+</p>
+
+<p>
+Be aware that external processes often store the standard out and
+standard error streams of the application, causing logged sensitive
+information to be stored.
```

Comment:
```suggestion
Be aware that external processes often store the standard output and
standard error streams of the application. This will include logged sensitive information.
```

'Standard out' -> 'Standard output' suggestion and minor: breaks sentence up slightly.